pkg/providers/instancetype/suite_test.go

```diff
@@ -740,7 +741,7 @@ var _ = Describe("InstanceTypes", func() {
 		Expect(err).To(BeNil())
 
 		for _, info := range instanceInfo {
-			it := instancetype.NewInstanceType(ctx, info, nodePool.Spec.Template.Spec.Kubelet, fake.DefaultRegion, windowsNodeClass, nil)
+			it := newInstanceType(nodePool, windowsNodeClass, info)
```

Comment:
IMO: It's better to call the function that we are actually testing in the test directly rather than wrapping it with a helper function